Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Test/TestIPAddressAC.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using System;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses.Test
+{
+    public class TestIPAddressAC
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'TestIPAddressAC' to match camel case naming rules, consider using 'TestIpAddressAC'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS101)